src/EnergyPlus/HeatBalanceKivaManager.cc

```diff
@@ -651,6 +652,12 @@ bool KivaManager::setupKivaInstances(EnergyPlusData &state)
     Kiva::setMessageCallback(kivaErrorCallback, nullptr);
     bool ErrorsFound = false;
 
+    // Need to process People objects before getting air setpoints for thermal comfort calculations
+    if (state.dataInternalHeatGains->GetInternalHeatGainsInputFlag) {
```

Comment:
OK, so this is just called once while setting up Kiva instances.  That's fine, although it feels like 20% intrusive.  Could we have just moved the call to `setupKivaInstances` to a little later in the simulation calling tree so that the internal gains are already done?  There's probably a reason why not but worth asking I think.

Reply:
I felt uneasy about it too. I think it is probably okay to move it after `ManageInternalHeatGains` in HeatBalanceManager. It will be a bit out of place (not with the other surface heat balance set up), but as long as there's a comment explaining it, I think it should be fine.

@matthew-larson do you want to give that a try?

Reply:
I'll try it out.

Reply:
It appears that this will work.  Doing some additional testing and rewriting the unit test to make sure it all checks out.